playbooks/setup.yml

```diff
@@ -0,0 +1,21 @@
+--- # Playbook for the initial setup of a new server before installing main packages
+
+# Ensure python is set up and accessible on the server before starting (ansible requires python2.7)
+- name: python check
+  hosts: ofn_servers
+  gather_facts: no
+  become: yes
+  tasks:
+    - name: Install python 2.7
+      raw: apt-get update -qq && apt-get install -qq python2.7
+      tags: skip_ansible_lint
+
+# Add the default user as root. AWS ubuntu images generally have a
+# default user (ubuntu) and wont let this be run as root. But digital ocean images don't.
+- name: set up default_user
+  hosts: ofn_servers
+  remote_user: root
```

Comment:
You are assuming `root` again. But not all images come with root access. Can we set root only if no remote user is defined?

Reply:
I havn't changed this role, just moved it. 

I think in this case where we don't have our default user yet, if the image doesn't allow root user access or has an alternate root user name, we can pass a different value for `remote_user` in the command line, with `-e "remote_user=ubuntu"` for example.

So it can already be overridden in those cases. Is there a more satisfying way of doing this?

Reply:
@sauloperez Does this work for you? Your solution was to take the root user out in https://github.com/openfoodfoundation/ofn-install/pull/219/files.